lib/core/src/Cardano/Wallet.hs

```diff
@@ -1493,45 +1493,6 @@ listTransactions ctx wid mStart mEnd order = db & \DBLayer{..} -> do
         _ ->
             pure $ slotRangeFromTimeRange sp $ Range mStart mEnd
 
-    -- This relies on DB.readTxHistory returning all necessary transactions
-    -- to assemble coin selection information for outgoing payments.
-    -- To reliably provide this information, it should be looked up when
-    -- applying blocks, but that is future work (issue #573).
-    assemble
```

Comment:
no longer needed, the db now returns `[TransactionInfo]` directly.

Reply:
Is it still noted somewhere that coin selection information can't (yet) be provided for outgoing payments?

Reply:
I've rewritten the comment and linked to the github issue in the relevant place in the db layer :+1: , good point